app/services/location_suggestion.rb

```diff
@@ -16,7 +16,7 @@ def initialize(location_input)
 
   def suggest_locations
     predictions = get_suggestions_from_google['predictions'].take(NUMBER_OF_SUGGESTIONS)
-    suggestions = predictions.map { |prediction| prediction['description'] }
+    suggestions = predictions.map { |prediction| prediction['description'].gsub(/, UK$/, '') }
```

Comment:
I was reviewing this as it got merged. We shouldn't do this and instead add `region: uk` to the settings